qtawesome/tests/test_icon_browser.py

```diff
@@ -56,24 +56,34 @@ def test_copy(qtbot, browser):
     # Click the copy button
     qtbot.mouseClick(browser._copyButton, QtCore.Qt.LeftButton)
 
-    assert "google" in clipboard.text()
+    assert "penguin" in clipboard.text()
 
-
-def test_filter(qtbot, browser):
+def test_filter_success(qtbot, browser):
     """
-    Ensure the filter UX works
+    What happens if a `penguin`
```

Comment:
Lets rewrite this to:

```suggestion
    Ensure the filter UX works when searching for `penguin`
```